lib/logstash/inputs/gelf.rb

```diff
@@ -33,6 +33,15 @@ class LogStash::Inputs::Gelf < LogStash::Inputs::Base
   # ports) may require root to use.
   config :port, :validate => :number, :default => 12201
 
+  # The GELF protocol (TCP or UDP).
+  config :protocol, :validate => [ "UDP", "TCP" ], :default => "UDP"
```

Comment:
Could we support both protocols simultaneously instead of having this be a setting?

For example, the `syslog` input listens on both tcp and udp at the same time.

Reply:
Mentioning that on doc would be great :)